clients/drcachesim/analyzer_multi.cpp

```diff
@@ -422,7 +422,8 @@ analyzer_multi_tmpl_t<RecordType, ReaderType>::analyzer_multi_tmpl_t()
                 nullptr, op_verbose.get_value(), op_jobs.get_value(),
```

Comment:
Does the invariant checker report discontinuities in syscall traces?  I remember the checker was relaxed/changed in some ways for syscall traces.  If the flag allowing gaps is turned on, should the user expect the checker to report errors?  That should be mentioned and documented.

Reply:
Yes it does. We just have a relaxation for the kernel_syscall_pt_trace test-mode in unit tests.